org.eclipse.xtext.tests/src/org/eclipse/xtext/xtext/ValidationIssue1034Test.xtend

```diff
@@ -0,0 +1,82 @@
+/*******************************************************************************
+ * Copyright (c) 2019 itemis AG (http://www.itemis.eu) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *******************************************************************************/
+package org.eclipse.xtext.xtext
+
+import com.google.inject.Inject
+import org.eclipse.xtext.Grammar
+import org.eclipse.xtext.XtextStandaloneSetup
+import org.eclipse.xtext.testing.InjectWith
+import org.eclipse.xtext.testing.XtextRunner
+import org.eclipse.xtext.testing.util.ParseHelper
+import org.eclipse.xtext.testing.validation.ValidationTestHelper
+import org.eclipse.xtext.tests.AbstractXtextTests
+import org.eclipse.xtext.tests.XtextInjectorProvider
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+import static org.eclipse.xtext.XtextPackage.Literals.PARSER_RULE
+import static org.eclipse.xtext.xtext.XtextConfigurableIssueCodes.INVALID_FRAGMENT_AS_FIRST_RULE
+
+/**
+ * @author Eva Poell - Initial contribution and API
+ */
+@RunWith(XtextRunner)
+@InjectWith(XtextInjectorProvider)
+class ValidationIssue1034Test extends AbstractXtextTests {
+
+	@Inject extension ParseHelper<Grammar>
+	@Inject extension ValidationTestHelper
+
+	@Before
+	override setUp() throws Exception {
+		super.setUp();
+		with(XtextStandaloneSetup)
```

Comment:
Is the with-invocation necessary?

Reply:
It seems to me like it.
The tests fail without that line the tests fail.